include/view/movement.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef PROC_GEN_OPENGL_MOVEMENT_HPP
+#define PROC_GEN_OPENGL_MOVEMENT_HPP
+
+
+#include <SDL_events.h>
```

Comment:
As I wrote to you in dm, not sure why it is not "#include <SDL2/SDL_events.h>"

Reply:
Done